docs/index.md

```diff
@@ -866,13 +904,77 @@ executed again.
 child_index: Child number ${{child_index}}
 ```
 
-#### `id`
+#### `unique_id`
 
-The `id` variable returns a unique identifier for the current Object/Row to allow you to construct unique identifiers.
+The `unique_id` variable returns a unique number that can
+be used to distinguish any record from others. For example,
+we can incorporate a unique ID into an email address or
+an employee ID.
 
 ```yaml
-fields:
-  name: ${{fake.LastName}} Household ${{id}}
+# examples/unique_id/test_unique_id.yml
+- object: Contact
+  count: 3
+  fields:
+    FirstName:
+      fake: FirstName
+    LastName:
+      fake: LastName
+    EmployeeNum: ${{unique_id}}
+    Email: ${{unique_id}}_${{fake.email}}
```

Comment:
@prescod is `fake` shared for FirstName and LastName Object instantiations? How does it know what email to get?

Reply:
The email is faked too. 

But yes it does have some internal heuristics which try to make the email match the firstname and lastname based on shared state attached to the Snowfakery context, however.